### PR TITLE
Grant package:write permission for UT image build

### PR DESCRIPTION
**Pull request by isarkis**
This is required to have new image pushed to GHR.

### CODE REVIEW

**Review by oxve — APPROVED** [review]

File: .github/workflows/main.yaml [hunk 1]
```diff
@@ -128,7 +128,8 @@ jobs:
   docker-unittest-image:
```
Comment (kaidokert) [inline .github/workflows/main.yaml:128]: Codespell check fails because "neccessary ==> necessary" change in the file above, could you address this here ?

**Review by oxve — CHANGES_REQUESTED** [review]
Apparently your change triggered the linter on main.yaml:

```
Spell Check..............................................................Failed
- hook id: codespell
- exit code: 65

.github/workflows/main.yaml:200: neccessary ==> necessary
```

File: .github/workflows/main.yaml (hunk 1 above)
Comment (isarkis) [reply .github/workflows/main.yaml:128]: done

**Comment by isarkis** [comment]
> Apparently your change triggered the linter on main.yaml:
> 
> ```
> Spell Check..............................................................Failed
> - hook id: codespell
> - exit code: 65
> 
> .github/workflows/main.yaml:200: neccessary ==> necessary
> ```

done

**Review by oxve — APPROVED** [review]

**Pull request merged**